test/extended/olmv1/olmv1.go

```diff
@@ -0,0 +1,60 @@
+package operatorsv1
+
+import (
+	"fmt"
+
+	g "github.com/onsi/ginkgo/v2"
+	o "github.com/onsi/gomega"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+)
+
+const (
+	operatorsGroupName = "olm.operatorframework.io"
+)
+
+var _ = g.Describe("[sig-olmv1] OLMv1 CRDs", func() {
+	defer g.GinkgoRecover()
+	oc := exutil.NewCLIWithoutNamespace("default")
+
+	g.It("should be installed", func(ctx g.SpecContext) {
+		// Check for tech preview, if this is not tech preview, bail
+		if !exutil.IsTechPreviewNoUpgrade(ctx, oc.AdminConfigClient()) {
+			g.Skip("Test only runs in tech-preview")
+		}
+
+		// supports multiple versions during transision
+		providedAPIs := []struct {
+			group   string
+			version []string
+			plural  string
+		}{
+			{
+				group:   operatorsGroupName,
+				version: []string{"v1alpha1", "v1"},
+				plural:  "clusterextensions",
+			},
+			{
+				group:   operatorsGroupName,
+				version: []string{"v1alpha1", "v1"},
+				plural:  "clustercatalogs",
+			},
+		}
+
+		for _, api := range providedAPIs {
+			g.By(fmt.Sprintf("checking %s at version %s [apigroup:%s]", api.plural, api.version, api.group))
+			// Ensure expected version exists in spec.versions and is both served and stored
+			var err error
+			var raw string
+			for _, ver := range api.version {
+				raw, err = oc.AsAdmin().Run("get").Args("crds", fmt.Sprintf("%s.%s", api.plural, api.group), fmt.Sprintf("-o=jsonpath={.spec.versions[?(@.name==\"%s\")]}", ver)).Output()
```

Comment:
nit: to avoid having to use escaped quotation marks

```suggestion
				raw, err = oc.AsAdmin().Run("get").Args("crds", fmt.Sprintf("%s.%s", api.plural, api.group), fmt.Sprintf(`-o=jsonpath={.spec.versions[?(@.name==%q)]}`, ver)).Output()
```

Reply:
This is a first pass at getting *something* in. The plan is to actually do a happy-path, but I had been waiting on other work to complete before I could get happy-path working.